### PR TITLE
Responsive workfolio: Desktop version

### PR DESCRIPTION
**Pull request by AbayomiOlaoye**
## Highlights
This PR features:
- Use of media queries to smoothen user experience across all devices (screen sizes)
- Manipulating webpage content with the use of media queries (max-width: 767px and min-width: 768px)
- Reducing form input element type of text to just one as per recommendation from microverse's study guide

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -216,7 +221,7 @@ <h2 class="recent-works robo-head center-align">Contact us</h2>
           <input type="text" class="full" name="fullname" id="fullname" placeholder="Full Name" maxlength="30" required><br>
           <input type="email" class="full" name="email" id="email" placeholder="youremail@example.com" required><br>
           <textarea id="textarea" class="full" name="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea><br>
-          <button type="submit" class="view robo-head">Get in touch</button>
+          <button type="submit" class="view robo-head"><span class="alt-del">Get in touch</span> <span class="hide">Send</span></button>
           <div class="error-index"></div>
         </form>
       </div>
```
Comment (mwanawabangona) [inline index.html]: - [x] Good job so far 👍 
- [x] However there is just a tiny 🤏  detail you missed on the contact form.
- [ ] the contact form on the name section its suppose to be separated the `first` and `last name` alone
- [ ] please see screenshots below

`figma template`
image    `Your template` 
contact form

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @MindPrepDev ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits
* ✅  Good Use of flexbox


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

**Review by mgmediaweb — APPROVED** [review]
Hi @MindPrepDev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved4

## Highlights  💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AbayomiOlaoye) [reply index.html]: @mgmediaweb Thank you for this commendation... I feel like you just handed me a trophy. Thanks, really!

**Pull request merged**